src/lib/registration/clients/scc_auto.rb

```diff
@@ -198,7 +197,16 @@ def online_medium_config
         Yast::Packages.ImportGPGKeys
 
         products = Y2Packager::ProductControlProduct.products
-        ay_product = Yast::Profile.current.fetch("software", {}).fetch("products", []).first
+
+        Yast.import "AutoinstFunctions"
```

Comment:
Um, this one is part of the `autoyast2-installation` package which we do not `Require`. But as this is only called during (auto)installation then it is safe. But maybe it is still worth adding a comment so it's not accidentally copied elsewhere where it might not work.

Reply:
yeah, it is already used in import. So I just copy it also there.